aws-opentelemetry-distro/src/amazon/opentelemetry/distro/exporter/otlp/aws/logs/otlp_aws_logs_exporter.py

```diff
@@ -34,3 +60,119 @@ def __init__(
             compression=Compression.Gzip,
             session=AwsAuthSession(aws_region=self._aws_region, service="logs"),
         )
+
+    def export(self, batch: Sequence[LogData]) -> LogExportResult:
+        """
+        Exports log batch with AWS-specific enhancements over the base OTLPLogExporter.
+
+        Key differences from upstream OTLPLogExporter:
+        1. Respects Retry-After header from server responses for proper throttling
+        2. Treats HTTP 429 (Too Many Requests) as a retryable exception
+        3. Always compresses data with gzip before sending
```

Comment:
Where is it documented that gzip is required for logs? This doc says both gzip and no compression are acceptable.
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-OTLPEndpoint.html#CloudWatch-LimitsandRestrictions